.vscode/tasks.json.sample

```diff
@@ -57,11 +57,20 @@
       "problemMatcher": [],
       "isBackground": true
     },
+    {
+      "label": "dev/ensure-admin",
+      "type": "shell",
+      "command": "bin/rails",
+      "args":["runner", "User.create!(username: \"admin\", password: \"DiscourseAdminPassword\", email: \"admin@example.com\", admin: true).activate if User.human_users.count == 0"],
```

Comment:
This is for development environments only, and I'd like to include the password in the docs so that people can get started quickly, without needing to run manual console commands.

Reply:
On second thoughts... i removed this for now. You're right that there is some risk, especially since some people make their dev environments internet-accessible